data/hai missions.txt

```diff
@@ -435,3 +435,585 @@ mission "Unfettered returning home"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	destination "Allhome"
+	to offer
+		has "event: hai in cargo"
+	on offer
+		conversation
+			`The "Check Cargo" light for your ship begins blinking when you land, meaning that a cargo crate may have come loose in flight. You shut off the ship's engines and walk to the cargo hold.`
+			`	You check every cargo crate in the hold, but can't find anything wrong. Right as you decide to leave, one of the smaller crates begins rocking back and forth.`
+			branch translator
+				has "Human Cultural Archives: done"
+				
+			`	You hear a voice come from the crate, but it is not a language you understand. The voice continues to speak, and you recognize it as the Hai language.`
+			choice
+				`	(Open the crate.)`
+			`	You open the crate and are met face to face with a Hai child. You aren't sure how old it is, but it looks to be about the height of a one year old human child. It tries to communicate with you, but begins to cry when it sees the confused look on your face.`
+			choice
+				`	"Don't cry, little guy."`
+					goto cry
+				`	(Pick up the Hai.)`
+				
+			`	The child waves its hands around when you try to pick it up, making it impossible to grab on to. After waiting for a few minutes, the child finally calms down, eventually going to sleep in the crate.`
+				goto end
+			
+			label cry
+			`	The child begins to cry even louder than before. After waiting for a few minutes, the child finally calms down, eventually going to sleep in the crate.`
+				goto end
+			
+			label translator
+			`	You hear a voice come from the crate, and surprisingly your Hai translator begins speaking. "Let me out! I want to go home!"`
+			choice
+				`	(Open the crate.)`
+			`	You open the crate and are met eye to eye with a Hai child. You aren't sure how old it is, but it looks to be about the height of a one year old human child. "I want my mommy!" it says as its eyes begin to tear up.`
+			choice
+				`	"Where's your mommy?"`
+				`	"Don't cry, little guy."`
+			`	The child looks surprised when your translation device speaks to it in Hai. "Take me home!" it yells back at you. You attempt to find out where the child's home is, but it continues to cry for its mother without giving any important information.`
+			
+			label end
+			`	This child must have somehow gotten into your cargo hold while you were in Hai space. Leaving a Hai child on a human world would be an extremely bad idea. Perhaps someone on <destination> will be able to bring the child back to its family.`
+				accept
+			
+	on complete
+		payment 50000
+		dialog `You contact the spaceport authorities of <planet> and ask them if they are looking for a lost Hai child. It turns out that the family was so worried that they put up a <payment> reward for finding their son. A Hai spaceport worker takes the child from your ship and thanks you for finding him for the family.`
```

Comment:
I'm not sure how Hai society works, but I can tell you that no human parent would allow a random "spaceport worker" to accept their child from the person who'd rescued him; they'd be there themselves, or wold want a registered social worker or someone vetted by the authorities to make the transfert if they couldn't be there in person.

Likewise, in this situation, a normal human would be extremely reluctant to hand over a child to a "spaceport worker"; the risk of a failed transfer or even the possibility of human (Hai?) trafficking would be seen as too high. Rather, the good samaritan would want to make the hand-off to the parents themselves, or to some kind of official child-welfare-caretaker-type personnel.

Reply:
The Hai are typically depicted as being more [innocent and naive](https://github.com/endless-sky/endless-sky/wiki/Story-Ideas#hai-diplomacy), so I'd think they'd be fine in having the child be handed off to a spaceport worker who would then make sure that the child gets where he/she needs to be, and given the efficiency of Hai society I'm certain our little Hai child here isn't going to get lost in transfer.

Reply:
Perhaps this part of the mission could be an opportunity to highlight those attributes, then, rather than requiring their assumption on the part of the player.

Reply:
Any specific suggested change to the dialog?